include/odp/api/spec/ml_types.h

```diff
@@ -329,6 +332,14 @@ typedef struct odp_ml_capability_t {
 
 /** Machine learning configuration parameters */
 typedef struct odp_ml_config_t {
+	/**
+	 * Engine ID to be configured.
+	 *
+	 * In a system with multiple ML engines, this parameter selects the engine to be configured.
+	 * The default value is 0.
+	 */
+	uint32_t engine_id;
```

Comment:
odp_ml_config() is called only once per ODP instance / application. This field/spec change would limit application to use only a single engine, which I think is not the purpose of the PR. Either this config field would need to be "maximum number of engines" or odp_ml_config() would need to be called once per engine (to be used).

In general, ODP API engine concept could/should support engines with different capacity/capability (e.g. two engines: "large" and "small"). That would require (new) capability call per engine.

Reply:
This change would enable odp_ml_config() to be called once per engine.

max_engines field has been added to odp_ml_capability to fetch the number of engines available.

Reply:
Updated the odp_ml_config() function documentation.